src/sentry/replays/consumers/recording/process_recording.py

```diff
@@ -162,6 +163,7 @@ def submit(self, message: Message[KafkaPayload]) -> None:
         # TODO: validate schema against json schema?
         try:
             message_dict = msgpack.unpackb(message.payload.value)
+            self._configure_sentry_scope(message_dict, message.offset)
```

Comment:
Should we only configure if an exception is thrown?

Reply:
i don't think its a big deal to do it here, and it's harder to do later on because we don't have the message_dict parsed.